apollo-router/src/plugins/telemetry/config.rs

```diff
@@ -56,12 +56,26 @@ pub(crate) struct Conf {
     /// Logging configuration
     #[serde(rename = "experimental_logging", default)]
     pub(crate) logging: Logging,
+
+    #[cfg(feature = "telemetry_next")]
+    #[serde(rename = "logging", default)]
+    pub(crate) new_logging: config_new::logging::Logging,
     /// Metrics configuration
     pub(crate) metrics: Metrics,
     /// Tracing configuration
     pub(crate) tracing: Tracing,
     /// Apollo reporting configuration
     pub(crate) apollo: apollo::Config,
+
+    #[cfg(feature = "telemetry_next")]
+    /// Event configuration
+    pub(crate) events: config_new::events::Events,
+    #[cfg(feature = "telemetry_next")]
+    /// Span configuration
+    pub(crate) spans: config_new::spans::Spans,
```

Comment:
as spans are only related to tracing why don't we put that in `tracing` section ?

Reply:
The reason that they are separate is that span data does end up in logs. Events also make it into traces, so that's the reason why both are separate.